app/admin/RTMediaFormHandler.php

```diff
@@ -513,6 +571,63 @@ static function general_content() {
 		self::render_tab_content( $render_options, $general_group, 90 );
 	}
 
+	/**
+	 * render export import.
+	 *
+	 * @access static
+	 *
+	 * @return array $render
+	 */
+	static function render_export_import() {
+		$render = array(
+			'rtmedia_export_settings' => array(
+				'title'               => esc_html__( 'Export rtMedia Settings', 'buddypress-media' ),
+				'callback'            => array( 'RTMediaFormHandler', 'button' ),
+				'args'                => array(
+					'id'    => 'rtm-export-button',
+					'key'   => 'rtmedia_export_settings',
+					'value' => esc_html__( 'Export Settings', 'buddypress-media' ),
+					'desc'  => esc_html__( 'This will export rtMedia settings into a JSON file.', 'buddypress-media' ),
+					'class' => array( 'button button-primary button-small' ),
```

Comment:
You need to pass classes as each array item.

`'class' => array( 'button', 'button-primary', 'button-small' ),`